src/db.ts

```diff
@@ -300,10 +299,10 @@ export abstract class Database {
     },
 
     fetchMaxMeasurements: `SELECT
-        runId, criterion, invocation as inv, max(iteration) as ite
+        runId, criterion, invocation as inv, cardinality(value) as ite
       FROM Measurement
       WHERE trialId = $1
-      GROUP BY runId, criterion, invocation
+      GROUP BY runId, criterion, invocation, ite
```

Comment:
hm, how come this is now grouped by `ite`?

Reply:
@HumphreyHCB sorry, I think we briefly discuss this, but I forgot, what was the explanation for this? Was this needed?

Reply:
It’s a yes and no. its needed as more of a placeholder for a method that calls it. But we never use the result of the ite column, I think in the record measurement method we now work out what the iteration should be.
If you remove it something will break, I am sure there is a way around this